typescript/scripts/README.adoc

```diff
@@ -0,0 +1,38 @@
+:toc: macro
+
+= tBTC v2 Refund script
+
+This script explains what to do in case you need to refund your BTC.
+
+toc::[]
+
+== Refund script
+
+=== Prerequisites
+
+Please make sure you have the following prerequisites installed on your machine:
+
+- https://nodejs.org[Node.js] >14.18.0
+- https://yarnpkg.com[Yarn] >1.22.19
+
+tBTCv2 system prerequisite is that you can refund your BTC only after `refundLocktime` expires, which is currently set to 9 month.
+
+=== Running a refund script
+
+Step 1. Clone https://github.com/keep-network/tbtc-v2[tbtc-v2] repository to your local disc. `git clone https://github.com/keep-network/tbtc-v2`
+
+Step 2. Navigate to a root directory and run the following command with your arguments:
+```
+./typescript/scripts/refund.sh --deposit-json-path <deposit-JSON-file-path> --amount <amount> --transaction-id <transaction-id> --private-key <wallet-private-key> --transaction-fee <transaction-fee> --electrum-host <electrum-host> --electrum-port <electrum-port> --electrum-protocol <electrum-protocol>
```

Comment:
Perhaps `--private-key <refunder-private-key>` instead of `--private-key <wallet-private-key>` ?
If we use `wallet` here, somebody may think it's a wallet that is meant to handle deposits (sweep them).

Reply:
sure thing, will do!